config.json

```diff
@@ -0,0 +1,153 @@
+﻿{
+  "settings": {
+    "OSGroup": {
+      "description": "Target OS for result binaries.",
```

Comment:
Thanks! I changed it. 
I'm not sure if we have documentation about it. I'm basically taking the values from the dir.props file